daemon/src/routes_maker.rs

```diff
@@ -106,3 +110,32 @@ pub fn get_health_check() {}
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 struct RetrieveCurrentOrder;
+
+#[derive(RustEmbed)]
+#[folder = "../frontend/dist/maker"]
+struct Asset;
+
+#[rocket::get("/assets/<file..>")]
+pub fn dist<'r>(file: PathBuf) -> impl Responder<'r, 'static> {
+    let filename = format!("assets/{}", file.display().to_string());
+    Asset::get(&filename).map_or_else(
+        || Err(Status::NotFound),
+        |d| {
+            let ext = file
+                .as_path()
+                .extension()
+                .and_then(OsStr::to_str)
+                .ok_or_else(|| Status::new(400))?;
+            let content_type = ContentType::from_extension(ext).ok_or_else(|| Status::new(400))?;
+            Ok((content_type, d.data))
+        },
+    )
+}
+
+#[rocket::get("/<_paths..>")]
+pub fn index<'r>(_paths: PathBuf) -> impl Responder<'r, 'static> {
+    Asset::get("index.html").map_or_else(
+        || Err(Status::NotFound),
+        |d| Ok((ContentType::HTML, d.data)),
```

Comment:
I'd find this easier to read (untested but I think should work):

```suggestion
    let asset = Asset::get("index.html").ok_or(Status::NotFound)?;
    
    Ok((ContentType::HTML, asset.data))
```

Or if you wanted to use one of rocket's built-in responders:

```rust
Ok(Html(asset.data))
```